linux/sample_driver.sh

```diff
@@ -9,11 +9,11 @@ DIRECTORY="/tmp/csvs"
 
 echo "Directory: $DIRECTORY."
 
-# Loop through every CSV file in the directory and call sendcsv.bat
+# Loop through every CSV file in the directory and call send_csv.sh
 DIRECTORY="$DIRECTORY/*.csv"
 for f in $DIRECTORY; do
 	echo "Uploading: $f."
-	./sendcsv.sh "$f"
+	./send_csv.sh "$f"
```

Comment:
why change to snake case? it looks like the file is integration/linux/sendcsv.sh